apps/chrome-devtools/project.json

```diff
@@ -165,7 +165,8 @@
       "options": {
         "eslintConfig": "apps/chrome-devtools/.eslintrc.js",
         "lintFilePatterns": [
-          "apps/chrome-devtools/src/**/*.ts"
+          "apps/chrome-devtools/src/**/*.ts",
+          "apps/chrome-devtools/package.json"
```

Comment:
Dummy question: Why not adding the project.json, too ?

Reply:
Today we have no rule applied to the `project.json` (or any `.json` generally, only to the specific `package.json`)
cf: https://nx.dev/packages/eslint-plugin/documents/dependency-checks

Reply:
ok :) thanks for details